### ISSUE TITLE
Clear form error messages upon value updating

### ISSUE THREAD

**Issue by markmhendrickson** [blockstack/blockstack-app#3856]
If I paste an invalid address into the Bitcoin send form then click elsewhere, it properly shows the relevant error message. But when I modify the value so it's valid, the error message doesn't go away until I click elsewhere again. 

It seems like we should clear the error message immediately after the user modifies the relevant input again, since the new value may be indeed correct but the user continues to think the wallet views it as invalid?

cc @mica000 

https://github.com/hirosystems/wallet/assets/28991/19f02e91-e33c-4844-b516-b829091b39c0

**Comment by kyranjamie** [comment]
Yep. The address field should validate on change (whereas amount field shouldn't).
